webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/index.js

```diff
@@ -7,10 +7,11 @@ import {
   Dropdown,
   KebabToggle,
 } from '@patternfly/react-core';
-import { foremanUrl } from '../../../../foreman_navigation';
+import { visit } from '../../../../foreman_navigation';
 import { translate as __ } from '../../../common/I18n';
+import { foremanUrl } from '../../../common/helpers';
 
-const ActionsBar = ({ hostName }) => {
+const ActionsBar = ({ hostName, permissions: { edit_hosts: canEdit } }) => {
```

Comment:
Should we skip permission check and allow edit when user is admin?

Reply:
I think `canEdit` will be true for admin

Reply:
Correct for the one that we seed, users can still create admins without actually assigning them any permissions.

Reply:
The permissions are deferred to [`Authorizer#can?`](https://github.com/theforeman/foreman/blob/7d12d02ec8be014284e5eeb102736d14f03f8c1d/app/views/api/v2/hosts/show.json.rabl#L19) so that should be fine for any admin: https://github.com/theforeman/foreman/blob/7d12d02ec8be014284e5eeb102736d14f03f8c1d/app/services/authorizer.rb#L15